src/JSInterop/Microsoft.JSInterop/src/Implementation/JSObjectReference.cs

```diff
@@ -2,56 +2,68 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
-using System.Text.Json;
 using System.Threading;
 using System.Threading.Tasks;
 
-namespace Microsoft.JSInterop
+namespace Microsoft.JSInterop.Implementation
 {
-    internal class JSObjectReference : IJSObjectReference
+    /// <summary>
+    /// Implements functionality for <see cref="IJSObjectReference"/>.
+    /// </summary>
+    public class JSObjectReference : IJSObjectReference
```

Comment:
@SteveSandersonMS and I discussed this change. We decided it was the best way to avoid giving IVT access to `M.A.Components.WebAssembly`. It follows a similar pattern as `JSRuntime`, which is also a public type. `JSObjectReference` is less visible in `M.JSInterop.Implementation`, and its constructor is `protected internal`, so customers aren't likely to use it by accident. Furthermore, [this](https://github.com/dotnet/aspnetcore/pull/25548/files#diff-2ba950045b5f06e6358c8cd052780120R9-R11) prevents developers from accidentally something like `JSRuntime.InvokeAsync<JSObjectReference>()`.